frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <v-container fluid>
+    <AppButton id="supp-allowances-button" size="large" width="250px" :to="{ name: 'supp-allowances' }" class="mb-12">Add SUP Application</AppButton>
```

Comment:
I know there are pending changes to this layout but can we update the name to match the mockup in the meantime?

Reply:
I think I follow the same name in this mockup: https://www.figma.com/file/PixIpiMff7mhG6D5SqMu3q/OFM-(New)?type=design&node-id=208-80&mode=design? maybe I look at the wrong one?

Reply:
Sorry about that. I thought I saw something else in the demo.